.github/workflows/ci.yml

```diff
@@ -1199,6 +1199,96 @@ jobs:
         timeout-minutes: 5
         run: cat /tmp/flv_sc.log
 
+  mirroring_smoke_test_e2e:
+    runs-on: ${{ matrix.os }}
+    needs:
+      - build_image
+      - config
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        rust-target: [x86_64-unknown-linux-musl]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup BATS
+        uses: mig4/setup-bats@v1
+        with:
+          bats-version: ${{ env.BATS_VERSION }}
+
+      # Download artifacts from development build
+      - name: Download artifact - fluvio
+        uses: actions/download-artifact@v4
+        with:
+          name: fluvio-${{ matrix.rust-target }}
+          path: ~/bin
+      - name: Download artifact - fluvio-run
+        uses: actions/download-artifact@v4
+        with:
+          name: fluvio-run-${{ matrix.rust-target }}
+          path: ~/extensions
+      - name: Download artifact - fluvio-test
+        uses: actions/download-artifact@v4
+        with:
+          name: fluvio-test-${{ matrix.rust-target }}
+          path: ~/extensions
+      - name: Download Docker Image as Artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: infinyon-fluvio-${{ matrix.rust-target }}.tar
+          path: /tmp
+
+      - name: Set up Fluvio binaries
+        run: |
+          chmod +x ~/bin/fluvio
+          chmod +x ~/extensions/fluvio-run
+          chmod +x ~/extensions/fluvio-test
+          mkdir -p ~/.fluvio/bin
+          mkdir -p ~/.fluvio/extensions
+          mv ~/bin/fluvio ~/.fluvio/bin/fluvio
+          mv ~/extensions/fluvio-run ~/.fluvio/bin/fluvio-run
+          mv ~/extensions/fluvio-test ~/.fluvio/bin/fluvio-test
+          echo "~/.fluvio/bin" >> $GITHUB_PATH
+
+      - name: Print version
+        run: fluvio version && fluvio-test -h
+
+      - name: Load Fluvio Docker Image
+        run: |
+          ls -la /tmp
+          docker image load --input /tmp/infinyon-fluvio-${{ matrix.rust-target }}.tar
+          docker image tag infinyon/fluvio:${{ github.sha }}-${{ matrix.rust-target }} infinyon/fluvio:${{ github.sha }}
+          docker image ls -a
+
+      - name: Export Docker Image to tarball
```

Comment:
why need to save image to tar again?

Reply:
is doing it twice?

Reply:
yes, no need to save image since it's already exported

Reply:
I got this from others test cases.
 https://github.com/infinyon/fluvio/blob/dcd3df0e4a85db4ef47ecbf646e0d78bb758e0ff/.github/workflows/ci.yml#L898

At line 898 it's saying that it's loading and exporting again to remove the arch from the tag.

This makes sense? Do we need this for this case?

Reply:
ah ok.  we should consolidate these as script but it's out of scope of this.